### ISSUE TITLE
Cant sneak and left click.

### ISSUE THREAD

**Issue by RavageMyRiven** [fr1kin/ForgeHax#393]
running 1.16.5 forge, but i cant sneak and left click, i can only sneak and right click. So i cant sneak and break blocks, or attack.

**Comment by RavageMyRiven** [comment]
i found out it only doesnt work when aimbot is on.

**Comment by fr1kin** [comment]
Yeah its related to some bizarre bug in minecraft where having anything bound to a mouse button at startup causes weird behavior
